src/common/containers/GameArea.js

```diff
@@ -234,7 +238,11 @@ export class GameArea extends Component {
     super(props);
 
     if (!props.started) {
-      this.props.history.push('/play');
+      if (props.history.location.pathname.includes('/tutorial')) {
```

Comment:
The `/tutorial` and `/play` string constants should probably be extracted from this location.

Reply:
`props.history.location.pathname.includes` is also a bit wordy. Maybe a wrapper of `history` is in order, could be more concise as `props.history.hasPart('/tutorial')` or something similar.

Reply:
@john-patterson How about now? I made things a bit more explicit.